app/app.gradle.kts

```diff
@@ -18,6 +18,7 @@ dependencies {
     compile(group = "com.google.code.gson", name = "gson", version = "2.8.1")
     fun testFx(name: String, version: String = "4.0.+") =
         create(group = "org.testfx", name = name, version = version)
+    runtime(group = "edu.wpi.first.ntcore", name = "ntcore-jni", version = "+", classifier = "all")
```

Comment:
GRIP isn't build with the plugins, and is on its own release schedule. I feel like this will end up getting published with the plugins, and I don't know if anyone will have time to change the release pipeline just to support a static version before beta.